zowe-rest-api-commons-spring/src/main/java/org/zowe/commons/error/ErrorServiceImpl.java

```diff
@@ -235,8 +218,7 @@ private String getLocalizedComponentOrUseDefault(Locale locale, String key, Erro
         String component = message.getComponent();
         if (component == null) {
```

Comment:
```suggestion
        String component;
        if ( message.getComponent() == null) {
```

I guess it is better just check and set the right value, component could be final, it is better to optimize.